File: SiteVisitsNetCore2020Web/ClientApp/src/app/models/Referrer.ts
export class Referrer {
  //id: number;
  id: string;
    cameFrom: string;
  shortCameFrom: string;
  constructor(options: {
    //id: number,
    id: string,
    cameFrom: string,
    shortCameFrom: string
  }) {
    if (options) {
      this.id = options.id;
      this.cameFrom = options.cameFrom;
      this.shortCameFrom = options.shortCameFrom;
    }
  }
}

File: SiteVisitsNetCore2020Web/Models/Referrer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SiteVisitsNetCore2020Web.Models
{
    public class Referrer
    {
        public int Id { get; set; }
        public string CameFrom { get; set; }
        public string ShortCameFrom { get; set; }
        public DateTime FirstOccurrence { get; set; }
        public DateTime DateCreated { get; set; }
    }
}
